src/Stratis.Bitcoin.Features.PoA/PoABlockHeader.cs

```diff
@@ -0,0 +1,52 @@
+﻿using NBitcoin;
+using NBitcoin.Crypto;
+
+namespace Stratis.Bitcoin.Features.PoA
+{
+    public class PoABlockHeader : BlockHeader
+    {
+        private BlockSignature blockSignature;
+
+        public BlockSignature BlockSignature
+        {
+            get => this.blockSignature;
```

Comment:
Can we use `public BlockSignature BlockSignature { get; set; }`?

Reply:
no, for the same reason as above- need this `ref` for btc stream